orttraining/orttraining/core/optimizer/sce_loss_grad_bias_fusion.cc

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "orttraining/core/optimizer/sce_loss_grad_bias_fusion.h"
+
+#include "core/graph/graph_utils.h"
+#include "core/optimizer/initializer.h"
+#include "core/optimizer/utils.h"
+
+using namespace ONNX_NAMESPACE;
+using namespace ::onnxruntime::common;
+
+namespace onnxruntime {
+
+Status SceLossGradBiasFusion::ApplyImpl(Graph& graph, bool& modified, int graph_level,
```

Comment:
the fusion graph is looks like this? 

SoftmaxCrossEntropyLossInternalGrad --> optional Reshape --> Add|Sum ?

Reply:
Right